getter.go

```diff
@@ -45,35 +42,73 @@ type getter struct {
 	cIdx int64
 }
 
-type chunk struct {
-	id     int
-	header http.Header
-	start  int64
-	size   int64
-	b      []byte
+type singleGetter struct {
+	getter
+	url        url.URL
+	contentLen int64
+	qWait      map[int]*Chunk
+}
+
+//New struct to hold methods that only make sense for multi-file downloads
+type batchGetter struct {
+	getter
+}
+
+type Chunk struct {
+	Id         int // The chunk number for the file being retrieved
```

Comment:
I agree these are better informative var names than what was there before (b, err etc...) but we will need to probably change them back before we push a pull request to the 3rd party repo